### ISSUE TITLE
Hercules does not exit when a config file does not exist. 

### ISSUE THREAD

**Issue by fbi-ranger** [SDL-Hercules-390/hyperion#164]
During testing I face the following behavior:

Starting hercules with a config file that can not be found, in this case because a slash for the path is missing, Hercules starts with following messages and does then nothing. It seems not to loop but wait on an interrupt. (Ctrl -C). Only then does it returns to command line: 

```
hercules -f etc/herctest.cnf
HHC00109E set_thread_priority( 5 ) failed: Operation not permitted
HHC00007I Previous message from function 'impl' at impl.c(837)
HHC00110W Defaulting all threads to priority 1
HHC00007I Previous message from function 'impl' at impl.c(840)
HHC00100I Thread id 00007ff74099c740, prio -1, name 'impl_thread' started
HHC00100I Thread id 00007ff73e050700, prio -1, name 'logger_thread' started
HHC02342S Configuration file 'etc/herctest.cnf' not found:  No such file or directory
HHC00007I Previous message from function 'process_args' at impl.c(1596)
HHC02343S Terminating due to 1 argument errors
HHC00007I Previous message from function 'impl' at impl.c(867)
^C
```

**Comment by Fish-Git** [comment]
Let me look into it!

**Comment by Fish-Git** [comment]
Fixed by commit eb73519dd42e0419fb3b7f642f6c867de2b0cf15.
